src/Requests/CompanyRequest.php

```diff
@@ -136,4 +137,16 @@ public function member(int $id): MemberRequest
             $id
         );
     }
+
+    /**
+     * Get the banned members in the company.
+     *
+     * @return BanIndexRequest
+     */
+    public function bans(): BanIndexRequest
+    {
+        return new BanIndexRequest(
+            $this->id
+        );
+    }
```

Comment:
This almost suggests like the company has bans. I think it should be a part of the `MemberIndexRequest` class.

Reply:
Good call.

I can migrate it to the `MemberIndexRequest` file and rename the method to `bans`, or just rename the method to `getBannedMembers` while not moving it. Do you have a preference, or any other ideas?

Reply:
Chaining methods in a way like `$client->company(1)->members()->bans()->get()` sounds better to me.  
If, however, it is not preferred (as it seems like it would be the first method in an index request class), then `bannedMembers()` sounds better. It just returns the members, not the bans.

@bensherred, what do you think? 👀

Reply:
I agree, chaining it like that indeed makes more sense for the method's purpose. I pushed an implementation to the above 👌 

You can now return a collection of the banned members within the company with `$client->company(1)->members()->bans()->get()`.